From: Tarek Soliman <tarek@bashasoliman.com>
Date: Mon, 4 Apr 2011 09:23:53 -0500
Subject: [PATCH] ALSA: usb-audio: define another USB ID for a buggy USB MIDI cable

commit 49c039f071d36586ba32da75996ef339e4ab8405 upstream.

There are many USB MIDI cables out there that have buggy
firmware that reports it can do more than 4 bytes in a
packet when they can only properly handle 4

This patch adds the ID of yet another one of those cables

Signed-off-by: Tarek Soliman <tarek@bashasoliman.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
[bwh: Adjust file name and line numbers for 2.6.32]
---
--- a/sound/usb/usbmidi.c
+++ b/sound/usb/usbmidi.c
@@ -1138,6 +1138,7 @@ static int snd_usbmidi_out_endpoint_create(struct snd_usb_midi* umidi,
 	case USB_ID(0x15ca, 0x0101): /* Textech USB Midi Cable */
 	case USB_ID(0x15ca, 0x1806): /* Textech USB Midi Cable */
 	case USB_ID(0x1a86, 0x752d): /* QinHeng CH345 "USB2.0-MIDI" */
+	case USB_ID(0xfc08, 0x0101): /* Unknown vendor Cable */
 		ep->max_transfer = 4;
 		break;
 		/*
